src-vue/src/plugins/store.ts

```diff
@@ -13,6 +13,7 @@ export const store = createStore({
             current_tab: Tabs.PLAY,
             developer_mode: false,
             game_path: "this/is/the/game/path",
+            install_type: undefined,
```

Comment:
Is this an enumeration?
I would like to see all values it can take :)

Reply:
Enumerations are defined in backend: https://github.com/GeckoEidechse/FlightCore/blob/93cec9e9cf8d1f67c38e4194b6c912aff06a3f22/src-tauri/src/lib.rs#L14-L19

Current upstream frontend simply uses a string that resembles the specific enumeration: https://github.com/GeckoEidechse/FlightCore/blob/93cec9e9cf8d1f67c38e4194b6c912aff06a3f22/src-ui/src/main.ts#L20
(mostly cause I didn't know how to do enums in TS yet :P)

Reply:
I think we should mirror the backend enum in TS.
Here's an enum example: https://github.com/Alystrasz/FlightCore/blob/feat/new-ui/src-vue/src/utils/Tabs.ts

Reply:
Added enum with a17edbf0cc2da4997048dd4196e7bfb3d8d2b21f ^^

Reply:
Just add a comment referencing the Rust file, and we're good to go!

Reply:
Done with 52394a3e86373801f5285843d6931a692a619be1